models/Location.v1.yaml

```
@@ -1,18 +1,20 @@
 title: Location
 description: "The root of the Location type's schema."
-required:
-  - id
-  - name
 type: object
+x-examples:
+  default:
+    id: 1
+    name: Default
+    max_capacity: 30
 properties:
   id:
     format: int32
     type: integer
   name:
     type: string
-x-examples:
-  default: |-
-    {
-      "id": 1,
-      "name": "Default"
-    }
\ No newline at end of file
+  max_capacity:
+    type: integer
+    description: Maximum capacity of a location
```

This field will need to explicitly allow `null` values:

```suggestion
  max_capacity:
    type: integer
    nullable: true
    description: Maximum capacity of a location
```